http/openbsd-5.6/install.conf

```diff
@@ -0,0 +1,11 @@
+System hostname = openbsd56
+Password for root = vagrant
+Do you expect to run the X Window System = no
+Setup a user = vagrant
+Password for user = vagrant
+What timezone are you in = UTC
+Location of sets = http
+HTTP proxy URL = none
+HTTP server = ftp5.usa.openbsd.org
+Server directory = pub/OpenBSD/5.6/i386
+Set name(s) = -game*.tgz -x*.tgz
```

Comment:
note that, while it is not an issue for your use case, X distributions are often required, even on simple routers, for graphic libraries and Java applications. https://github.com/reallyenglish/packer-templates/pull/12/commits/310fac29b730cca38644bf8c87a0768e07513972